rfcs/text/0014-extend-file-pe.md

```diff
@@ -77,17 +78,217 @@ Stage 1: Provide a high-level description of example sources of data. This does
 
 <!--
 Stage 2: Included a real world example source document. Ideally this example comes from the source(s) identified in stage 1. If not, it should replace them. The goal here is to validate the utility of these field changes in the context of a real world example. Format with the source name as a ### header and the example document in a GitHub code block with json formatting.
+https[://]c4a08b8258a848efb518dcea18a1d11e[.]us-central1[.]gcp[.]cloud[.]es[.]io/goto/51d883b52d4ad99d9200c1b25cf1b5c9
 -->
-
+```
+"file": {
+  "pe": {
+    "machine_type": 332,
+    "flattened": {
+      "debug": [
+        {
+          "codeview": {
+            "signature": "RSDS",
+            "name": "G:\\Work\\new\\2345Pinyin\\bin\\Win32\\Release\\pdb\\2345PinyinSymbol.pdb",
+            "guid": "c27b9b78-caab-4629-9bf7-6acb4b63ac18",
+            "age": 1
+          },
+          "offset": 350324,
+          "size": 90,
+          "type_str": "IMAGE_DEBUG_TYPE_CODEVIEW",
+          "type": 2,
+          "timestamp": "Tue Jan 26 09:13:07 2021"
+        }
+      ],
+      "resources": [
+        {
+          "chi2": 257629.96875,
+          "filetype": "Data",
+          "entropy": 3.32720947265625,
+          "sha256": "49ba440033219598d7a9e5e27f5185063634c379dfaf4b188e49f0f43572beef",
+          "type": "RT_ICON",
+          "lang": "CHINESE SIMPLIFIED"
+        }
+      ]
+    },
+    "file_version": "7.4.0.8215",
+    "product": "2345王牌输入法",
+    "imports": [
+      {
+        "name": "GetStdHandle",
+        "library_name": "KERNEL32.dll",
+        "type": "function"
+      }
+    ],
+    "comments": "pinyin.2345.cc",
```

Comment:
I don't see `comments` in the proposed extended `pe.*` fields or the current ones. Is this a field to add?

Reply:
@dcode were the comments added from your dataset or is that part of the PE file?

Reply:
Questions caused by stale data examples. Refactoring.